### ISSUE TITLE
Customize Keyboard Shortcuts

### ISSUE THREAD

**Issue by mdroidian** [zsviczian/obsidian-excalidraw-plugin#162]
**Is your feature request related to a problem? Please describe.**
I use CTRL-Shift-[ and CTRL-Shift-] for `Fold / Unfold all headings and lists`.
When I try CTRL-Shift-[ or CTRL-Shift-] in an Excalidraw drawing, there is no effect. 

**Describe the solution you'd like**
If this cannot be avoided (IE CTRL-Shift-[ working for BOTH `fold all headings and lists` AND `Send To Back`), then the ability to customize the Excalidraw shortcuts (like `Send To Back`) would be awesome.

**Additional context**
I believe this may require this https://github.com/excalidraw/excalidraw/issues/3706

**Comment by zsviczian** [comment]
I agree this is something that needs to be worked on, and you are right, Excalidraw native shortcuts depend on the core product. I don’t promise that I’ll be able to develop a solution for this any time soon though.

**Comment by jcyh0120** [comment]
+1 for this.

**Comment by nttldnknwn** [comment]
+1, #6275

**Comment by AhmedHawater2003** [comment]
+1

**Comment by zsviczian** [comment]
I appreciate the +1s, but please add your +1 on the Excalidraw issue log, not here. I will not be doing anything with this issue as it needs to be implemented on Excalidraw.com

in the meantime, you can create Excalidraw scripts and assign Obsidian hotkeys to create your own shortcuts. It is pretty simple to do. No coding skills are required.

https://youtu.be/6BjhUyfS4iM

**Comment by zsviczian** [comment]
regarding the original issue - you can add your Obsidian hotkey overrides in Excalidraw plugin settings.

<img width="1050" height="435" alt="Image" src="https://github.com/user-attachments/assets/2f6034c5-ebbc-4f6e-b243-98470b32f3bf" />
